compiler-core/templates/prelude.js

```diff
@@ -130,6 +130,14 @@ export class BitString {
   get length() {
     return this.buffer.length;
   }
+
+  atLeastLength(desired) {
+    return desired <= this.length;
```

Comment:
Can we inline these functions in the generated code rather than add these methods? I added them for the list classes mostly just because they're too complicated to inline nicely

Reply:
sure, I created those methods only after I spotted them on the list for similarity.